crates/bevy_ecs/src/schedule/condition.rs

```diff
@@ -142,7 +142,7 @@ pub mod common_conditions {
 
     /// Generates a [`Condition`](super::Condition)-satisfying closure that returns `true`
     /// if the first time the condition is run and false every time after
-    pub fn run_once() -> impl FnMut() -> bool {
+    pub fn run_once() -> impl Clone + FnMut() -> bool {
```

Comment:
readability nit: FnMut is more important here than Clone so IMO it should come first.

Reply:
`FnMut() -> bool + Clone` looks weird to me. It makes it look like the function is returning `bool + Clone` somehow.